smarty/templates/main.tpl

```diff
@@ -155,6 +175,7 @@
     </head>
     {/if}
     <body>
+    <div id="test_name" style="display:none">{$test_name}</div>
```

Comment:
It's bad practice to use hidden DOM elements as your data model, it's slow and doesn't scale.. but also irrelevant if this is changed to be a format=csv query string since the server will just return the csv directly and this javascript isn't required..